base-pom/pom.xml

```diff
@@ -231,6 +273,21 @@
         <version>0.4.2</version>
       </dependency>
 
+      <!-- JTidy hasn't been released for ages (since 2001). A snapshot version exists at
+           http://jtidy.sourceforge.net/snapshots/ but Maven requires non-snapshots in order to
+           allow a release (this is for reproducibility). Thus we've had to upload JTidy into
+           our custom repository -->
+      <dependency>
+        <groupId>jtidy</groupId>
```

Comment:
is there something wrong with the new version @msladek added? Including two different versions is probably not a good idea.

Reply:
The versions were mixed up in the 5.0.1 and the 5.2-SNAPSHOT version of the cleverreach component pom-file. I just extracted both to the dependency version management in the base-pom file. THUS please tell me which one is no the correct one?

Reply:
I found the reason, how it came to. to `      <groupId>jtidy</groupId>
      <artifactId>jtidy</artifactId>
` is used in celements-core and the  `        <groupId>com.github.jtidy</groupId>
        <artifactId>jtidy</artifactId>` is used in celements-cleverreach. I updated to dependency in celements-core and removed the old version im base-pom.